packages/cpp/src/browser/cpp-task-provider.ts

```diff
@@ -37,10 +38,33 @@ export class CppTaskProvider implements TaskContribution, TaskProvider, TaskReso
     @inject(TaskResolverRegistry) protected readonly taskResolverRegistry: TaskResolverRegistry;
     @inject(TaskDefinitionRegistry) protected readonly taskDefinitionRegistry: TaskDefinitionRegistry;
     @inject(CppBuildConfigurationManager) protected readonly cppBuildConfigurationManager: CppBuildConfigurationManager;
+    @inject(ProblemMatcherRegistry) protected readonly problemMatcherRegistry: ProblemMatcherRegistry;
+    @inject(ProblemPatternRegistry) protected readonly problemPatternRegistry: ProblemPatternRegistry;
 
     @postConstruct()
     protected init(): void {
         this.registerTaskDefinition();
+        this.problemPatternRegistry.register({
+            name: 'clangTidyPattern',
+            regexp: '^(.+):(\\d+):(\\d+):\\s+(error|warning|info|note):\\s+(.+?)\\s+\\[(.+)\\]$',
```

Comment:
Please be consistent with adding quotations.

Reply:
Done